### ISSUE TITLE
Question about the SIDD raw results.

### ISSUE THREAD

**Issue by leonmakise** [swz30/CycleISP#20]
Can CycleISP reach 52.41dB PSNR on SIDD benchmark or just on validation set? 

Even with the pretrained model, I cannot get that result on the offical server.
